components/data_prefetch/index.ts

```diff
@@ -75,8 +77,9 @@ function mapStateToProps(state: GlobalState) {
 
 function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
     return {
-        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, Actions>({
+        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, any>({
```

Comment:
Won't block this PR but is it possible to add the correct type?

Reply:
Since `Actions` was updated above, this looks like it could use the correct type instead of `any`